packages/nouns-contracts/contracts/NounsArt.sol

```diff
@@ -0,0 +1,450 @@
+// SPDX-License-Identifier: GPL-3.0
+
+/// @title The Nouns art storage contract
+
+/*********************************
+ * ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ *
+ * ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ *
+ * ░░░░░░█████████░░█████████░░░ *
+ * ░░░░░░██░░░████░░██░░░████░░░ *
+ * ░░██████░░░████████░░░████░░░ *
+ * ░░██░░██░░░████░░██░░░████░░░ *
+ * ░░██░░██░░░████░░██░░░████░░░ *
+ * ░░░░░░█████████░░█████████░░░ *
+ * ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ *
+ * ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ *
+ *********************************/
+
+pragma solidity ^0.8.6;
+
+import { INounsArt } from './interfaces/INounsArt.sol';
+import { SSTORE2 } from './libs/SSTORE2.sol';
+import { IInflator } from './interfaces/IInflator.sol';
+
+contract NounsArt is INounsArt {
```

Comment:
@solimander agreed, we discussed this today as well. We think it still is valuable for the art adding functions to go through descriptor because that's where we will implement the part retirement logic and state. However for stuff like setting the inflator address, the executor can totally be the admin.

We felt at this stage of the project it's not worth it to create a new role, but definitely something we can improve in the next version of NounsArt.

makes sense?

Reply:
Good point on the trait retirement stuff. Makes sense 👍